CondFormats/SiPixelObjects/BuildFile.xml

```diff
@@ -1,20 +1,28 @@
 <use name="alpaka"/>
-<use name="DataFormats/DetId"/>
-<use name="DataFormats/SiPixelDetId"/>
 <use name="CalibFormats/SiPixelObjects"/>
-<use name="FWCore/Utilities"/>
-<use name="DataFormats/FEDRawData"/>
-<use name="CondFormats/Serialization"/>
+<use name="CondCore/DBOutputService"/>
 <use name="CondFormats/DataRecord"/>
-<use name="CondFormats/SiStripObjects"/>
-<use name="Geometry/TrackerGeometryBuilder"/>
 <use name="CondFormats/External"/>
+<use name="CondFormats/Serialization"/>
+<use name="CondFormats/SiStripObjects"/>
+<use name="DataFormats/DetId"/>
+<use name="DataFormats/FEDRawData"/>
 <use name="DataFormats/GeometryVector"/>
+<use name="DataFormats/Portable"/>
+<use name="DataFormats/SiPixelDetId"/>
+<use name="DataFormats/SoATemplate"/>
+<use name="DataFormats/TrackerCommon"/>
+<use name="FWCore/Framework"/>
 <use name="FWCore/MessageLogger"/>
+<use name="FWCore/ParameterSet"/>
+<use name="FWCore/ServiceRegistry"/>
+<use name="FWCore/Utilities"/>
 <use name="Geometry/CommonDetUnit"/>
 <use name="Geometry/CommonTopologies"/>
-<use name="HeterogeneousCore/AlpakaInterface"/>
+<use name="Geometry/Records"/>
```

Comment:
The dependence on `Geometry/Records` should also be moved to `CondFormats/SiPixelObjects/test/BuildFile.xml`
```suggestion
```

Reply:
done